collect_app/src/main/java/org/odk/collect/android/forms/MediaFileRepository.java

```diff
@@ -1,9 +0,0 @@
-package org.odk.collect.android.forms;
-
-import java.io.File;
-import java.util.List;
-
-public interface MediaFileRepository {
```

Comment:
When I got into it, it really felt like removing this and switching to absolute paths being returned from `FormRepository` was a lot simpler than what we had.